src/iotjs_module_buffer.cpp

```diff
@@ -216,6 +235,10 @@ JHANDLER_FUNCTION(Write) {
 
   BufferWrap* buffer_wrap = BufferWrap::FromJBufferBuiltin(*jbuiltin);
 
+  offset = boundRange(offset, 0, buffer_wrap->length());
```

Comment:
Isn't `BoundRange(offset, 0, buffer_wrap->length() - 1);` the correct call of this function?